assert.lua

```diff
@@ -82,6 +82,20 @@ function assert:type_or_nil(a, t, msg, stack_level)
 	return a
 end
 
+--assert a value is one of those in a table of options
+function assert:one_of(a, t, msg, stack_level)
+	local pass = false
+	for index = 1, #t do
+		if t[index] == a then
+			pass = true
+			break
+		end
+	end
+
+	assert:equal(pass, true, msg, stack_level)
```

Comment:
this should just be the same as the actual assert code, because this reports the failure as within assert.lua :)

can also just use ipairs and return early if it's found (skipping an error outside the loop) for much more terse code.

Reply:
Sure thing, I mainly figured that a normal `for` loop would be faster, though.

Reply:
Should we also perhaps have an assert:truthy and assert:falsey or is that too much fluff?